src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -155,5 +155,18 @@ limitations under the License.
       <arg name="returnValue" type="INT8U"/>
     </command>
 
+    <command source="server" code="0x02" name="TestAddArrayofStructArguments" optional="false">
```

Comment:
What is this command adding?  The existing "TestAdd" command actually adds things up on the server, in the sense of arithmetic.   This needs a different name, unless we are adding up things in those structs.  Maybe "TestArrayofStructArguments"?

Also, why is source "server" here?  That seems odd.

Reply:
This should be "client" right?

Reply:
yep, would update in #10171